odoo/tools/sql.py

```diff
@@ -160,19 +189,23 @@ def __iter__(self):
 
     def join(self, args: Iterable) -> SQL:
         """ Join SQL objects or parameters with ``self`` as a separator. """
+        if not self:
+            return SQL(self, *args)
+        # handle no arguments and make args a list
+        empty = SQL()
```

Comment:
This variable is used twice in 2 different condition branches. It is better to repeat `SQL()` at both places.